core/node/api_server/src/tx_sender/mod.rs

```diff
@@ -234,6 +247,9 @@ pub struct TxSenderConfig {
     pub validation_computational_gas_limit: u32,
     pub chain_id: L2ChainId,
     pub whitelisted_tokens_for_aa: Vec<Address>,
+    pub timestamp_asserter_address: Option<Address>,
+    pub timestamp_asserter_min_range_sec: u32,
+    pub timestamp_asserter_min_time_till_end_sec: u32,
```

Comment:
Why not use `Option<TimestampAsserterParams>` here? Do `min_range_sec` / `min_time_till_end_sec` params still make sense even if the asserter address isn't set?

Reply:
Makes sense - changed